src/components/pipelines-overview/PipelineRunsNumbersChart.tsx

```diff
@@ -18,17 +18,67 @@ import {
   getXaxisValues,
   parsePrometheusDuration,
 } from './dateTime';
+import { useTRPipelineRuns } from '../hooks/useTektonResults';
+import { ALL_NAMESPACES_KEY } from '../../consts';
+import { DataType, TektonResultsOptions } from '../utils/tekton-results';
+// import { DataType, TektonResultsOptions } from '../utils/tekton-results';
 
 interface PipelinesRunsNumbersChartProps {
+  namespace?: string;
   timespan?: number;
+  interval?: number;
   domain?: DomainPropType;
+  pipelineName?: string;
```

Comment:
we should rename this prop to `parentName` and also add a `kind` prop so that it can be handled for the Repository metrics page.